lib/team_statistics.rb

```diff
@@ -0,0 +1,22 @@
+module TeamStatistics
+
+  def create_team_hash(team)
+    {"team_id"      => team.team_id,
+     "franchise_id" => team.franchiseid,
+     "team_name"    => team.teamname,
+     "abbreviation" => team.abbreviation,
+     "link"         => team.link}
+  end
+
+  def find_team(team_id)
+    teams.find do |team|
+      team if team.team_id == team_id
```

Comment:
I think you don't need the if here. The team.team_id == team_id condition should work just as well.

`def find_team(team_id)
    teams.find { |team| team.team_id == team_id }
end`

Reply:
Made the change.